Cargo.toml

```diff
@@ -24,6 +24,7 @@ members = [
     "wgpu-types",
     "wgpu",
     "xtask",
+    "naga/tests",
```

Comment:
Alphabetize please. Need to copy the auto-alphabetizing code from work to the taplo config...